README.md

```diff
@@ -29,7 +29,7 @@ Ensure Python 3.8+ is available on your machine (see [this guide](https://docs.p
 
 Install requirements in a Python venv
 ```bash
-python3.8 -m venv venv/
+python3.8 -m virtualenv venv/
```

Comment:
this intentionally uses `venv` l, which is built in, instead of `virtualenv`, which has to be installed